packages/lun/lun.0.0.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer:   [ "romain.calascibetta@gmail.com" ]
+authors:      "Romain Calascibetta"
+license:      "ISC"
+homepage:     "https://git.robur.io/robur/lun"
+bug-reports:  "https://git.robur.io/robur/lun/issues"
+dev-repo:     "git+https://github.com/roburio/lun.git"
+doc:          "https://roburio.github.io/lun/"
+synopsis:     "Optics in OCaml"
+description: """(▀̿Ĺ̯▀̿ ̿)"""
```

Comment:
Can we do a little better than this description? Bear in mind that this will show up on ocaml.org's documentation search, so a hint to users about what this package actually does is useful here (for example, extracted from the README)